op-guide/configuration.md

```diff
@@ -50,3 +50,152 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### $TIDB_PPROF environment variable
 + An environment variable that is used to enable or disable the runtime profiling data via the HTTP server. The Address is at client URL + "/debug/pprof/".
 + If set $TIDB_PPROF to 0, TiDB will disable pprof. Otherwise TiDB will enable pprof.
+
+## PD
+
+### -L
+
++ The log level
++ default: "info"
++ You can choose from debug, info, warn, error, or fatal.
+
+### -c 
+
++ The config file
++ default: ""
+
+### --cluster-id
+
++ The cluster ID to identify unique cluster. 
++ default: 0
++ You must use a unique ID to distinguish different clusters. 
+
+### name
+
++ The human-readable unique name for this PD member.
++ default: "pd"
++ If you want to start multiply PDs, you must use different name for each one.
+
+### --host
+
++ The server host IP. 
++ default: "127.0.0.1"
++ You must set this flag so that it can be connected from outside., and can only ignore this in standalone test. 
```

Comment:
You must set this flag so that it can be connected from outside. You can only ignore it in a standalone mode.